### ISSUE TITLE
Dealing with spam bots from the front end

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-feature-requests#616]
There is no way to easily deal with spambots or sometimes real people from spamming a section of the web site (which typically is the blog).  Most of these spam bots registers a bunch of accounts to do this. The feed that shows shows their spam posts and it usually a series of them one right after the other. 

Currently, the only way to deal with these unruly accounts is to log into the admincp and one by one search for the users, copy and paste their IP to the IP ban list and then delete the account. This is a nightmare.

An easier way to deal with this issue is if we can just select the all spam users from the feed status and have their IP automatically added to the IP ban list and then delete the accounts.

**Comment by neil96** [comment]
Hi @prepperz ,

You can enable user group setting `Enable CAPTCHA challenge when adding a new blog?`.

Regards.

**Comment by prepperz** [comment]
Yes you can, but that doesn't solve any solve any of the issues as described above which is to quickly select the users from the front end in the feed and have the IP added to the ban list and then delete accounts into one single process.
